BetaUtils.Core.Tests/Extensions/IEnumerableExtensionsTests.cs

```diff
@@ -0,0 +1,138 @@
+﻿using BetaUtils.Core.Extensions;
```

Comment:
Please recheck naming of methods as methods that are being tested have changed theirs functions (eg. IsNullOrEmpty_ReturnsFalse_WhenCollectionIsNotNullOrEmpty does not validate if its null or not) 💯